packages/cli/src/commands/microfrontends/command.ts

```diff
@@ -0,0 +1,39 @@
+import { packageName } from '../../util/pkg-name';
+
+export const pullSubcommand = {
+  name: 'pull',
+  aliases: [],
+  description: 'Pull a Vercel Microfrontend configuration into your project',
+  arguments: [],
+  options: [
+    {
+      name: 'dpl',
+      shorthand: null,
+      deprecated: false,
+      type: String,
+      description:
+        'The deploymentId to use for pulling the Microfrontend configuration',
+      required: true,
+    },
+  ],
```

Comment:
Would be nice if this also accepted deployment urls like other commands, but not necessary. Also, `required` here is not true, but shouldn't do anything for options anyway.

```suggestion
  options: [
    {
      name: 'dpl',
      shorthand: null,
      deprecated: false,
      type: String,
      description:
        'The deploymentId to use for pulling the Microfrontend configuration',
    },
  ],
```